connector/src/main/java/org/jboss/as/connector/subsystems/resourceadapters/Constants.java

```diff
@@ -434,6 +437,7 @@ public void marshallAsElement(AttributeDefinition attribute, ModelNode resourceM
             .setDefaultValue(new ModelNode(ELYTRON_MANAGED_SECURITY))
             .addAccessConstraint(ResourceAdaptersExtension.RA_SECURITY_DEF)
             .setNullSignificant(false)
+            .setDeprecated(ELYTRON_BY_DEFAULT_VERSION)
```

Comment:
This is already there a couple lines down.

Reply:
fixed